lua/wire/client/text_editor/texteditor.lua

```diff
@@ -2506,6 +2506,14 @@ function EDITOR:AC_Check( notimer )
 	self:AC_SetVisible( false )
 end
 
+local function concat_args(args_table, where, cend)
```

Comment:
Making this a function is unnecessary, make a table and push each argument and then concat that inside of the FindEvents function